templates/src/php/docker/download_phpunit.include

```diff
@@ -1,3 +1,4 @@
-RUN wget https://phar.phpunit.de/phpunit-4.8.36.phar && ${'\\'}
-  mv phpunit-4.8.36.phar /usr/local/bin/phpunit && ${'\\'}
+RUN wget https://phar.phpunit.de/phpunit-5.7.27.phar && ${'\\'}
+  mv phpunit-5.7.27.phar /usr/local/bin/phpunit && ${'\\'}
   chmod +x /usr/local/bin/phpunit
+  
```

Comment:
This is an extra line that haven't been reflected in all the generated file.

Reply:
got it

Reply:
Sorry, I forgot run generate_projects.sh to make file match after remove extra line